<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StyleTester</class>
 <widget class="qMRMLWidget" name="StyleTester">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>544</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="toolTip">
        <string>Load stylesheet from a file into the editor</string>
       </property>
       <property name="text">
        <string>Load Style Sheet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="toolTip">
        <string>Save contents of editor to file</string>
       </property>
       <property name="text">
        <string>Save Style Sheet</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="plainText">
      <string>QWidget{ color: #00ff00; border: 2px solid green}
QPushButton{border: 2px solid blue}</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="styleTargetLayout">
     <item>
      <widget class="QRadioButton" name="examplesRadio">
       <property name="toolTip">
        <string>Apply style only to example section below</string>
       </property>
       <property name="text">
        <string>Apply to examples</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">styleTargetGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="slicerRadio">
       <property name="toolTip">
        <string>Apply style to all of Slicer</string>
       </property>
       <property name="text">
        <string>Apply to Slicer</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">styleTargetGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Apply stylesheet in editor to the selected target</string>
       </property>
       <property name="text">
        <string>Apply Style</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="toolTip">
        <string>Clear style from the selected target</string>
       </property>
       <property name="text">
        <string>Clear Style</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="Examples">
     <property name="text">
      <string>Example Widgets</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="buttonCheckBox">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkCheckBox">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="spinCheckBox">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="spinBox"/>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="hsliderCheckBox">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSlider" name="horizontalSlider">
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="toolCheckBox">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="labelCheckBox">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="enabledLabel">
        <property name="toolTip">
         <string>Checkboxes control the enabled/disabled state of their corresponding widgets</string>
        </property>
        <property name="text">
         <string>Enabled?</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="widgetLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Widget</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="comboCheckBox">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>New Item 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>New Item 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>New Item 3</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QCheckBox" name="nodeCheckBox">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox"/>
      </item>
      <item row="10" column="0">
       <widget class="QCheckBox" name="radioCheckBox">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>RadioButton</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">exampleGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>RadioButton</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">exampleGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit">
        <property name="text">
         <string>LineEdit</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="lineEditCheckBox">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hsliderCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nodeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>MRMLNodeComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>radioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>radioButton_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StyleTester</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="styleTargetGroup"/>
  <buttongroup name="exampleGroup"/>
 </buttongroups>
</ui>
